seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -1167,71 +1278,32 @@
-        "Returning to the conga line example, a new person wants to join the line, but he wants to join in the middle. You are in the middle of the line, so you take your hands off of the person ahead of you. The new person walks over and puts his hands on the person you once had hands on, and you now have your hands on the new person.",
-        "Instructions",
-        "Create an addAt(index,element) method that adds an element at a given index. Return false if an element was unable to be added.",
-        "Note",
-        "Remember to check if the given index is a negative or is longer than the length of the linked list."
-      ],
-      "challengeSeed": [
-        "function LinkedList() { ",
-        "  var length = 0; ",
-        "  var head = null; ",
-        "",
-        "  var Node = function(element){",
-        "    this.element = element; ",
-        "    this.next = null; ",
-        "  }; ",
-        "",
-        "  this.size = function(){",
-        "    return length;",
-        "  };",
-        "",
-        "  this.head = function(){",
-        "    return head;",
-        "  };",
-        "",
-        "  this.add = function(element){",
-        "    var node = new Node(element);",
-        "    if(head === null){",
-        "        head = node;",
-        "    } else {",
-        "        currentNode = head;",
-        "",
-        "        while(currentNode.next){",
-        "            currentNode  = currentNode.next;",
-        "        }",
-        "",
-        "        currentNode.next = node;",
-        "    }",
-        "",
-        "    length++;",
-        "  }; ",
-        "",
-        "  // Only change code below this line",
-        "",
-        "  // Only change code above this line",
-        "",
-        "};"
-      ],
-      "tests": [
-        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); test.addAt(0,'cat'); return test.head().element === 'cat'}()), 'message: Your <code>addAt</code> method should reassign <code>head</code> to the new node when the given index is 0.');",
-        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); test.addAt(0,'cat'); return test.size() === 3}()), 'message: Your <code>addAt</code> method should increase the length of the linked list by one for each new node added to the linked list.');",
-        "assert((function(){var test = new LinkedList(); test.add('cat'); test.add('dog'); return (test.addAt(4,'cat') === false); }()), 'message: Your <code>addAt</code> method should return <code>false</code> if a node was unable to be added.');"
-      ],
-      "solutions": [],
-      "hints": [],
-      "type": "waypoint",
-      "challengeType": 1,
-      "translations": {}
+      "translations": {
```

Comment:
Please revert this line of change. We do not need to add empty translation blocks.